diff --git a/build/native-tabs/utils/icon.js b/build/native-tabs/utils/icon.js
index 0196ac206a1a7d6d56172bc92bbf87bccbde405f..c6fd4e15837d0926d51a14a13600cd111649e5a7 100644
--- a/build/native-tabs/utils/icon.js
+++ b/build/native-tabs/utils/icon.js
@@ -46,7 +46,7 @@ function convertOptionsIconToRNScreensPropsIcon(icon) {
         return undefined;
     }
     if ('sf' in icon && icon.sf) {
-        return { sfSymbolName: icon.sf };
+        return { ios: { type: 'sfSymbol', name: icon.sf } };
     }
     else if ('src' in icon && icon.src) {
         return { templateSource: icon.src };
